test/stores/filters_store_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:economiza_sc/models/market.dart';
+import 'package:economiza_sc/modules/app_module.dart';
+import 'package:economiza_sc/stores/filters_store.dart';
+import 'package:flutter_modular/flutter_modular.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mobx/mobx.dart' as mobx;
+import 'package:mockito/mockito.dart';
+
+void main() {
+  final market = Market(id: 1, name: '', website: '');
+
+  setUp(() {
+    Modular.init(AppModule());
+  });
+
+  tearDown(() {
+    Modular.destroy();
+  });
+
+  group('FilterStore class', () {
+    group('updateSearch method', () {
```

Comment:
Let's improve the description of this test by changing it too: when the action "updateSearch" is called, the search value is updated.

Reply:
Got it!